### ISSUE TITLE
Aphakia

### ISSUE THREAD

**Issue by daboe01** [obophenotype/human-phenotype-ontology#10902]
Preferred term label:
Aphakia

Synonyms
Absence of the lens, Lenslessness

Definition (free text, please give PubMed ID)
A condition characterized by the absence of the crystalline lens in the eye, which may be congenital or acquired (e.g., following surgical removal of the lens for cataracts or trauma).

Parent term (use hpo.jax.org/app)
Abnormal lens morphology (HP: 517)

Diseases characterized by this term
Congenital aphakia (HP: 707) but also Post-cataract surgery

Your nano-attribution (ORCID)
0000-0001-9042-0508

**Comment by daboe01** [comment]
i think it would be good to make a new parent term "Aphakia" in the global sense that there is no lens in the eye.
as child terms we could have "Congenital aphakia" (7707) and "Iatrogenic aphakia" (new)

**Comment by pnrobinson** [comment]
@daboe01 
adding
Adding new term: 
- ID: HP:6001302
- label: Aphakia
- parents: [Abnormal lens morphology (HP:0000517)](https://hpo.jax.org/app/browse/term/HP:0000517)
- pmid: 
- def: Absence of the crystalline lens in the eye, which may be congenital or acquired (e.g., following surgical removal of the lens for cataracts or trauma).
- comment: 
- orcid: orcid:0000-0001-9042-0508
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10902
- synonyms: 

Will add Iatrogenic as well

**Comment by pnrobinson** [comment]
@daboe01 
also added
Acquired aphakia
HP:0025111
def: Absence of the crystalline lens in the eye known to be the result of a non-congenital condition, e.g., following surgical removal of the lens for cataracts or trauma.
